code/drasil-example/gamephysics/lib/Drasil/GamePhysics/DesignDec/Vector.hs

```diff
@@ -166,17 +166,17 @@ vectUnrotate = funcDef "vectUnrotate" [vV1, vV2] (Vect Rational)
     )
   ]
   
-vectLengthSq = funcDef "vectLengthSq" [v_v] Rational
+vectLengthSq = funcDef "vectLengthSq" [v_v] Real
   [
     FRet (FCall (funcUID vectDot) [v_v, v_v])
   ]
   
-vectLength = funcDef "vectLength" [v_v] Rational
+vectLength = funcDef "vectLength" [v_v] Real
   [
     FRet (Sqrt (FCall (funcUID vectLengthSq) [v_v]))
   ]
```

Comment:
I don't quite understand why these are here. They all look like they should be explained somewhere outside of GamePhysics and imported here. Since they're all related to functionality of Vectors, they might really be part of a "standard library" that should be built for each example using vectors in any of its expressions.

Reply:
Completely agree - please open an Issue for this.